java/java.editor/test/unit/data/goldenfiles/org/netbeans/modules/editor/java/GoToSupportTest/javadocsnippet_HighlightAndReplace_cornercase.pass

```diff
@@ -0,0 +1,5 @@
+<html><body><font size='+0'><b><a href='*0'>test.&#x200B;Test</a></b></font><pre>public void <b>testHighlightAndReplace_cornercase</b>()</pre><p><pre><code>  public static void \bmain\b(String... args) {	
```

Comment:
This file is missing a license header.

Reply:
Addressed in latest commit(bb448a61294adecae9571d7d9e02279c15e7c1d9)